source/features/one-click-review-submission.tsx

```diff
@@ -68,9 +68,7 @@ function replaceCheckboxes(originalSubmitButton: HTMLButtonElement): void {
 	}
 
 	// Remove original fields at last to avoid leaving a broken form
-	for (const radio of radios) {
-		radio.closest('.form-checkbox')!.remove();
-	}
+	radios[0].closest('fieldset')!.remove();
```

Comment:
The radios are now wrapped in a `fieldset`, which also has padding of its own, so it's cleaner just to remove.

Accessing the first radio and therefore its closest `fieldset` felt like the most straightforward way to achieve without having to find anything more specific about the fieldset (it only has an `m-3` class)